customLabels.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"encoding/json"
+	go_cache "github.com/patrickmn/go-cache"
+	"strings"
+	"time"
+	plog "github.com/prometheus/common/log"
+)
+
+// Custom labels for ConsumerGroups
+type CustomCGLagLabels struct {
+	labelByPrefix			map[string]string
+	labelCache			*go_cache.Cache
+}
+
+func NewCustomCGLagLabels(config string) (*CustomCGLagLabels, error){
+	labelByOwner := make(map[string][]string)
+	err := json.Unmarshal([]byte(config), &labelByOwner)
+	if err != nil {
+		return nil, err
+	}
+
+	labelByPrefix := make(map[string]string)
+	for owner, startWith := range  labelByOwner{
+		for _, startWith := range startWith {
+			labelByPrefix[startWith] = owner
```

Comment:
should we fail if a label is defined for more hat 1 owner? or at least print a warning message that we are going to override that with the last entry

Reply:
a  warn log would be ok, not failing. it is a limitation that I talked with Pablo Diaz, they never should repeat the "StartWith" if so just the latest one read will be used as it becomes the "key" of the map. It worth to say that adding the same start with does not make any sense as the owner tag can be added once per metric. So it is true this could be made by mistake. I will add a warning log